instrumentation/rack/lib/opentelemetry/instrumentation/rack/instrumentation.rb

```diff
@@ -22,6 +22,14 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
           defined?(::Rack)
         end
 
+        option :allowed_request_headers,  default: [],    validate: ->(v) { v.is_a?(Array) }
+        option :allowed_response_headers, default: [],    validate: ->(v) { v.is_a?(Array) }
+        option :application,              default: nil,   validate: ->(v) { v.respond_to?(:call) }
+        option :record_frontend_span,     default: false, validate: ->(v) { v.is_a?(TrueClass) || v.is_a?(FalseClass) }
+        option :retain_middleware_names,  default: false, validate: ->(v) { v }
```

Comment:
What's the deciding factor for choosing
```ruby
default: false, validate: ->(v) { v.is_a?(TrueClass) || v.is_a?(FalseClass) }
```
vs
```ruby
default: false, validate: ->(v) { v }
```

Reply:
Ah, this was just some leftover.  Will make it consistent.